inference-engine/src/preprocessing/cpu_x86_avx512/ie_preprocess_gapi_kernels_avx512.hpp

```diff
@@ -180,27 +180,26 @@ void splitRow_32FC4(const float in[],
                           float out2[],
                           float out3[],
                             int length);
+}  // namespace avx512
 
-void calculate_nv12_to_rgb(const  uchar **srcY,
-                           const  uchar *srcUV,
-                                  uchar **dstRGBx,
-                                    int width);
 
-void calculate_i420_to_rgb(const  uchar **srcY,
-                           const  uchar *srcU,
-                           const  uchar *srcV,
-                                  uchar **dstRGBx,
-                                    int width);
+template<typename isa_tag_t, typename T>
+void chanToPlaneRowImpl(isa_tag_t, const T* in, const int chan, const int chs, T* out, const int length);
 
-void copyRow_8U(const uint8_t in[],
-                uint8_t out[],
-                int length);
+extern template void chanToPlaneRowImpl(avx512_tag, const uint8_t* in, const int chan, const int chs, uint8_t* out, const int length);
```

Comment:
Is `extern template` intended here? Just looking at [this](https://stackoverflow.com/questions/8130602/using-extern-template-c11).